src/testers.jl

```diff
@@ -28,25 +28,28 @@ Call `FiniteDifferences.j′vp`, with the option to ignore certain `xs`.
 - `∂xs::Tuple`: Derivatives estimated by finite differencing.
 """
 function _make_fdm_call(fdm, f, ȳ, xs, ignores)
-    sig = Expr(:tuple)
-    call = Expr(:call, f)
-    newxs = Any[]
-    arginds = Int[]
-    i = 1
-    for (x, ignore) in zip(xs, ignores)
-        if ignore
-            push!(call.args, x)
-        else
-            push!(call.args, Symbol(:x, i))
-            push!(sig.args, Symbol(:x, i))
-            push!(newxs, x)
-            push!(arginds, i)
+    ignores = collect(ignores)
+    function f2(sigargs...)
```

Comment:
I think we should use a better name than `f2` here.

Reply:
Sure! Have any ideas? (naming things is hard)

Reply:
Actually thinking about it some more, since we're already passing in `f` as the function to differentiate, I think `f2` is fine here.

Could you just make `f2` the first thing defined in this function? GitHub doesn't let me submit suggestions on deleted line. :/

```
function _make_fdm_call(...)
  function f2(sigargs...)
    ...
  end

  ignores = collect(ignores)
```